### PR TITLE
Added random C codes

### PR DESCRIPTION
**Pull request by VamsiSangam**

### CODE REVIEW

File: edit-distance.c [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+	for (i = 1; i <= aLength; ++i) {
+		for (j = 1; j <= bLength; ++j) {
+			if (a[i - 1] == b[j - 1]) {
+				dp[i][j] = dp[i - 1][j - 1];
+			} else {
+				dp[i][j] = 1 + min(
+								dp[i][j - 1],
+								dp[i - 1][j], 
+								dp[i - 1][j - 1]);
+			}
```
Comment (crimsoncowl) [inline edit-distance.c:44]: Doesnt this look odd?

**Comment by VamsiSangam** [comment]
No. Long lines are not very readable

File: edit-distance.c (hunk 1 above)
Comment (VamsiSangam) [reply edit-distance.c:44]: Nope

Comment (crimsoncowl) [reply edit-distance.c:44]: ok

**Review by crimsoncowl — APPROVED** [review]

**Pull request merged**